modules/gltf/src/lib/types/gltf-json-schema.ts

```diff
@@ -105,6 +105,8 @@ export interface Accessor {
   name?: any;
   extensions?: any;
   extras?: any;
+  value?: any;
```

Comment:
Is this truly `any`? Can we be more precise? `TypedArray`?

Reply:
removed it